decidim-participatory_processes/app/queries/decidim/participatory_processes/visible_participatory_process_groups.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # This query class filters participatory processes given a current_user.
+    class VisibleParticipatoryProcessGroups < Rectify::Query
+      def initialize(current_user)
+        @current_user = current_user
+      end
+
+      def query
+        processes = Decidim::ParticipatoryProcess.all
```

Comment:
Should this be scoped to the user's organization?

Reply:
This is scoped in other query `OrganizationParticipatoryProcessGroups`.